.github/workflows/semver_checks.yml

```diff
@@ -79,3 +82,14 @@ jobs:
       env:
         GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         GH_REPO: ${{ github.repository }}
+
+  semver-push-tag:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push'
+    timeout-minutes: 30
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install semver-checks
+      run: cargo install cargo-semver-checks --no-default-features
+    - name: Verify that's it's safe to publish the version.
```

Comment:
I think this label implies too much, that the tool does really exhaustive checks and its assessment about semver compatibility can be trusted.